### ISSUE TITLE
Add auto-format

### ISSUE THREAD

**Issue by pronebird** [ericmartel/Sublime-Text-2-CSV-Plugin#6]
Would be great to have auto format option, I have 20 columns, typing all of {0} ... is not productive. I just want to output CSV as table.

**Comment by ericmartel** [comment]
Hi,

When you say "as a table", what kind of table are we talking about?

You'd want something that auto generates {0} {1} {2} etc... and you fill the blanks between the numbers?

**Comment by pronebird** [comment]
@ericmartel indeed, automatically print out the entire CSV with spacer around so it's easier to read.

**Comment by ericmartel** [comment]
Done
https://github.com/ericmartel/Sublime-Text-2-CSV-Plugin/commit/7c7ac6472c4d3c01ffd906f7d797d4f943fbad8c#diff-930589f66b26168dad507a83f2a5975e

**Comment by ericmartel** [comment]
@pronebird Didn't mean to close, give me your feedback, if it's good enough ill close the issue

**Comment by pronebird** [comment]
Thanks so much!

**Comment by pronebird** [comment]
When I tap "format with default values" in command palette, I see the input box with the following:

`{0} {1} {2} {3} {4} {5} {6} {7} {8} {9} {10} {11} {12} {13} {14} {15} {16} {17} {18} {19} {20}{0} {1} {2} {3} {4} {5} {6} {7} {8} {9} {10} {11} {12} {13} {14} {15} {16} {17} {18} {19} {20}{0} {1} {2} {3} {4} {5} {6} {7} {8} {9} {10} {11} {12} {13} {14} {15} {16} {17} {18} {19} {20}{0} {1} {2} {3} {4} {5} {6} {7} {8} {9} {10} {11} {12} {13} {14} {15} {16} {17} {18} {19} {20}{0} {1} {2} {3} {4} {5} {6} {7} {8} {9} {10} {11} {12} {13} {14} {15} {16} {17} {18} {19} {20}{0} {1} {2} {3} {4} {5} {6} {7} {8} {9} {10} {11} {12} {13} {14} {15} {16} {17} {18} {19} {20}`

Also, fields are not aligned, so the formatted output is hard to follow.

**Comment by ericmartel** [comment]
How many columns do you have?  21?

Oh... I think I know the issue :) I tried it quickly and it worked, but I guess it's not reset between the times you use it

**Comment by ericmartel** [comment]
Sorry I should have tested thoroughly, I just pushed an update :)

**Comment by pronebird** [comment]
Works great, thanks so much! I've added pretty formatting in #7.
